### PR TITLE
texstudio: 3.0.1 -> 3.0.3

### PR DESCRIPTION
**Pull request by ajs124**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/texstudio/default.nix [hunk 1]
```diff
@@ -24,7 +24,8 @@ mkDerivation rec {
       LaTeX editing with completion, structure viewer, preview,
       spell checking and support of any compilation chain.
     '';
-    homepage = "http://texstudio.sourceforge.net";
+    homepage = "https://texstudio.org";
+    changelog = "https://github.com/texstudio-org/texstudio/blob/${version}/utilities/manual/CHANGELOG.txt";
     license = licenses.gpl2Plus;
     platforms = [ "x86_64-linux" ];
```
Comment (prusnak) [inline pkgs/applications/editors/texstudio/default.nix:30]: It seems this builds on macOS and my guess is it builds on aarch64-linux too  
```suggestion
    platforms = platforms.unix;
```

**Review by prusnak — COMMENTED** [review]
LGTM.

I think we can enable the build on other platforms as well (see comment)

File: pkgs/applications/editors/texstudio/default.nix (hunk 1 above)
Comment (ajs124) [reply pkgs/applications/editors/texstudio/default.nix:30]: Does it actually run on darwin?

Comment (prusnak) [reply pkgs/applications/editors/texstudio/default.nix:30]: Hm, the derivation got built, but did not contain any binaries in $out

Maybe let's just use `platforms.linux` then

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107904` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>texstudio</li>
  </ul>
</details>

**Pull request merged**